css/styles.css

```diff
@@ -98,20 +98,32 @@
   flex-basis: 45%;
 }
 
+.create-btn {
+  background-color: rgb(242, 242, 242);
+}
+.create-btn:hover {
+  background-color: rgb(242, 242, 242);
+}
```

Comment:
Suggestion: You can use variables for these colors

Reply:
Done, Thanks for pointing it out @Tarandeep-s1ngh